packages/amplication-server/src/core/workspace/workspace.service.ts

```diff
@@ -798,6 +798,55 @@ export class WorkspaceService {
     }
   }
 
+  async bulkUpdateWorkspaceProjectsAndResourcesLicensed(
```

Comment:
it would be great to have an idea of the performance of this bulkUpdate. Probably testing this in sandbox and checking the tracing data

Reply:
we will test it